library/Zend/Db/Adapter/Driver/Oci8/Oci8.php

```diff
@@ -130,6 +162,48 @@ public function getResultPrototype()
     }
 
     /**
+     * Add feature
+     *
+     * @param string $name
+     * @param AbstractFeature $feature
+     * @return Pdo
+     */
+    public function addFeature($name, $feature)
+    {
+        if ($feature instanceof AbstractFeature) {
+            $name = $feature->getName(); // overwrite the name, just in case
+            $feature->setDriver($this);
+        }
+        $this->features[$name] = $feature;
+        return $this;
+    }
+
+    /**
+     * Setup the default features for Pdo
+     *
+     * @return Pdo
```

Comment:
@samsonasik `@return $this` is recognized by phpDocumentor 2 and has a better meaning